### PR TITLE
feat: mainRow2 component added , screen width standards added

### PR DESCRIPTION
**Pull request by supreeth-spiger**
mainRow2 component added , screen width standards added

### CODE REVIEW

File: src/components/rows/MainRow2/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import PropTypes from "prop-types";
+import SliderMain from "../../atoms/SliderMain";
```
Comment (nandap4790) [inline src/components/rows/MainRow2/index.js:4]: let's group the third party imports together ,the same repo imports together and css together

File: src/components/rows/MainRow2/mainRow2.m.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  --background-color: var(--off-white);
+  --accent-color: var(--ecstasy);
+  background-color: var(--background-color);
+  padding: var(--space1_5) 0 var(--space3);
+  @media (min-width: 768px) {
+    padding: var(--space3) 0;
+  }
+}
+.main-row :global .widget300xauto {
+  background: none;
```
Comment (nandap4790) [inline src/components/rows/MainRow2/mainRow2.m.css]: all globals inside main-row

**Review by nandap4790 — APPROVED** [review]

File: src/components/rows/MainRow2/index.js (hunk 1 above)
Comment (supreeth-spiger) [reply src/components/rows/MainRow2/index.js:4]: done

File: src/components/rows/MainRow2/mainRow2.m.css (hunk 2 above)
Comment (supreeth-spiger) [reply src/components/rows/MainRow2/mainRow2.m.css]: done

**Pull request merged**